### PR TITLE
[Fixes] Added missing closing bracket to Bicep deployment example, removed unsued VM module parameter & regenerated documentation

### PR DESCRIPTION
**Pull request by MrMCake**
# Description

- Added missing closing bracket to Bicep deployment example, 
- Removed unsued VM module parameter `enableDefaultTelemetry`
- Regenerated documentation

| Pipeline |
| - |
| |

# Type of Change

Please delete options that are not relevant.

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

### CODE REVIEW

File: utilities/tools/Set-ModuleReadMe.ps1 [hunk 1]
```diff
@@ -452,6 +452,7 @@ function Set-DeploymentExamplesSection {
                 '  params: {'
                 ($bicepParamsArray | ForEach-Object { "  $_" }),
                 '  }'
+                '}'
```
Comment (eriqua) [inline utilities/tools/Set-ModuleReadMe.ps1:455]: I guess it needs to run on the newly added authorization locks as well. Could you do that?

Comment (MrMCake) [reply utilities/tools/Set-ModuleReadMe.ps1:455]: I sure can.

Comment (MrMCake) [reply utilities/tools/Set-ModuleReadMe.ps1:455]: done

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**